frontend/src/app/shared/components/modals/preview-modal/hover-card-modal/hover-card.modal.sass

```diff
@@ -7,11 +7,12 @@
     opacity: 1
 
 .op-hover-card
-  animation: fade-in-hover-card .25s ease-in-out
+  animation: fade-in-hover-card .25s ease-in
   position: absolute
   background-color: var(--body-background)
   z-index: 5000
   width: 500px
+  min-height: 95px
```

Comment:
since there is no loading indicator, I think there is no need to set a min-height for the hover card. What do you think @EinLama ?